Blog/templates/Blog/content.html

```diff
@@ -1,9 +1,18 @@
 {% extends 'Blog/base.html' %}
 {% block title %} Actual content {% endblock %}
 {% block pagecontent %}
+    <a href="{% url 'Blog:homepage' %}">Back To HomePage</a><br/><br/>
     <h4>Title:</h4>
     {{blog.title}}
     <h4>Content:</h4>
     {{blog.content}} <br/><br/>
-    <a href="{% url 'Blog:homepage' %}">Back To HomePage</a>
+    {%if messages %}
```

Comment:
Please put messages before the blog content. Users will miss it if they are forced to scroll a long post till the end